Homework 5.cpp

```diff
@@ -0,0 +1,161 @@
+//
+//  Homework 5.cpp
+//  StudyFall16
+//
+//  Created by Farida Eid on 7/8/17.
+//  Copyright © 2017 Farida Eid. All rights reserved.
+//
+
+#include <iostream>
+#include <vector>
+#include <unordered_map>
+#include <stack>
+
+using namespace std;
+
+class Graph
+{
+private:
+    int currentVertices;      // number of verticies
+    vector<vector<int>>* adjList;       // vector of edges
```

Comment:
No point to change it now, but I think it would have been easier to use a `map<char, set<char>> adjList` (so you avoid the bookkeeping with currentVertices, alphaIndex & indexAlpha)
(Also, why a pointer?)